assets/css/style.css

```diff
@@ -0,0 +1,77 @@
+
+body {
+    margin: 0;
+    overflow: hidden;
+    background-color: #000;
+    position: relative;
+}
+
+.wrapper {
+    width: 100%;
+    height: 100vh;
+    position: relative;
+    
+}
+
+#dvd-logo {
+    width: 300px;
+    height: 150px;
+    position: absolute;
+    top: 0;
+    left: 0;
+    display: block;
+}
+
+.myanimation {
+    animation-name: dvd-logo;
+    animation-duration: 2s;
+    animation-timing-function: ease-in-out;
+    animation-iteration-count: infinite;
+}
+
+
+.dvd-logo-img {
+    display: block;
+    width: 100%;
+}
+
+@keyframes dvd-logo {
+
+    0% {
+        transform: translate(0, 0);
```

Comment:
Moving around borders is boring :)
Think about how to make the movement quasi-chaotic (See Lissajous figures)

I like that you use calc(100vw - 300px)